estate/models/estate_property_offer.py

```diff
@@ -0,0 +1,64 @@
+from datetime import timedelta
+from odoo import api, models, fields
+from odoo.exceptions import UserError
+from odoo.tools import float_is_zero
+
+
+class EstatePropertyOffer(models.Model):
+    _name = "estate.property.offer"
+    _description = "Real Estate Property Offer"
+    _order = "price desc"
+
+    # Fields declaration
+    price = fields.Float("Price")
+    status = fields.Selection(
+        [("accepted", "Accepted"), ("refused", "Refused")], string="Status", copy=False
+    )
+    partner_id = fields.Many2one("res.partner", string="Partner", required=True)
+    property_id = fields.Many2one("estate.property", string="Property", required=True)
+    validity = fields.Integer("Validity (days)", default=7)
+    date_deadline = fields.Date(
+        "Deadline", compute="_compute_deadline", inverse="_inverse_deadline"
```

Comment:
Update method declaration accordingly
```suggestion
        "Deadline", compute="_compute_date_deadline", inverse="_inverse_date_deadline"
```